src/snapshot.js

```diff
@@ -222,9 +222,12 @@ export function restorePlayerSnapshot(player, callback) {
     player.one('contentloadedmetadata', restoreTracks);
 
     // adding autoplay guarantees that Safari will load the content so we can
-    // seek back to the correct time after ads
-    if (videojs.browser.IS_IOS && !player.autoplay()) {
-      player.autoplay(true);
+    // seek back to the correct time after ads.
+    // This is done directly on the html5 tech because if the integration has set
+    // normalizeAutoplay to true, the async play request via autoplay -> manualAutoplay_
+    // causes a visible skipback of the content after the ad break
+    if (videojs.browser.IS_IOS && !player.autoplay() && typeof player.techCall_ === 'function') {
+      player.techCall_('setAutoplay', true);
```

Comment:
So, if I understand correctly, setting it directly on the tech is synchronous and bypasses Video.js' built-in manual autoplay behavior?

Reply:
yes that's right.

A test or two would be good, I'll see if I can add something.